pulpcore/cli/core/pulpexport.py

```diff
@@ -0,0 +1,107 @@
+from typing import List
+import click
+from pulpcore.cli import main
+
+RepoIdentifier = click.Tuple([str, str])
+
+
+@main.group()
+@click.pass_context
+def pulpexporter(ctx):
```

Comment:
I started to adopt a different (more specific) pattern here. See core/task.py and #23 .

Reply:
Ah interesting - when #23 is merged, I will revisit and follow the trail you're blazing!